packages/stack/package.json

```diff
@@ -18,7 +18,7 @@
   "homepage": "https://stack-auth.com",
   "scripts": {
     "typecheck": "tsc --noEmit",
-    "build": "rimraf dist && pnpm run codegen && tsup-node",
+    "build": "rimraf dist && pnpm run css && tsup-node",
```

Comment:
Why?

Reply:
the Quetzel code gen breaks the docker build process. I think ideally we shouldn't do network requests during build

Reply:
Sounds good, but we should do it during CI/CD in the "Lint and Build" script so we get an error when the quetzal translations are out of date (it errs out when Git shows changes at the end of the build)